lib/pup/application.rb

```diff
@@ -1,9 +1,27 @@
 require "rack"
+require "pup/dependencies/string"
+require "pup/dependencies/object"
 
 module Pup
   class Application
-    def self.call(_env)
-      [200, {}, ["pup: A Ruby Framework for web masters"]]
+    attr_reader :request, :verb, :path
+    def call(env)
```

Comment:
Unused method argument - `env`. If it's necessary, use `_` or `_env` as an argument name to indicate that it won't be used. You can also write as `call(*)` if you want the method to accept any arguments but don't care about them.